File: src/sdk-reference/cpp/1/essentials/error-handling/snippets/error-handling.cpp
try {
  kuzzle->index->create("nyc-open-data");
} catch (kuzzleio::GatewayTimeoutException e) {
  /* Won't be called */
} catch (kuzzleio::BadRequestException e) {
  std::cout << "Status: " << e.status << " Message: " << e.getMessage() << std::endl;
  std::cout << "Try with an other name!" << std::endl;
} catch (kuzzleio::KuzzleException e) {
  /* Won't be called */
}

File: src/sdk-reference/go/1/essentials/error-handling/snippets/error-handling.go
// Type assertion of error to KuzzleError
err := kuzzle.Index.Create("nyc-open-data", nil)

if err != nil {
  fmt.Println(err.(types.KuzzleError).Message)

  if err.(types.KuzzleError).Status == 400 {
    fmt.Println("Try with an other name!")
  }
}
